File: app/src/main/java/com/android/MyApplication.java
package com.android;

import android.app.Application;

import com.android.frameworkkotlin.network.MyOkHttpClientKt;

/**
 * 当前类的注释:
 * 作者：WangLiJian on 2020/3/23.
 * 邮箱：wanglijian1214@gmail.com
 */
public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        MyOkHttpClientKt.initOkHttpClient();
    }
}

File: app/src/main/java/com/android/MyApplication.kt
package com.android

import android.app.Application
import android.content.Context
import kotlin.properties.Delegates

/**
 * 当前类的注释:application
 * 作者：WangLiJian on 2020/6/15.
 * 邮箱：wanglijian1214@gmail.com
 */


class MyApplication: Application() {
    companion object{
        var appContext:Context by Delegates.notNull()
    }
    override fun onCreate() {
        super.onCreate()
        appContext=this

    }
}